app/controllers/questions_controller.rb

```diff
@@ -0,0 +1,46 @@
+class QuestionsController < ApplicationController
+  def index
+    @questions = Question.all
+  end    
+
+  def show
+    @question = Question.find(params[:id])
+    
+    if @question.resolved 
```

Comment:
I read on stack that usually we do not code in .erb files. However , I tried to checkout into the assignment and ended up in the wrong branch. After I fixed everything I had to merge to master. Thats ok however I also made the changes in view .